src/user/schema.graphql

```diff
@@ -69,7 +69,7 @@ type InstructorProfile {
 	"""
 	The papers and publications selected
 	"""
-	selectedPapersAndPublications: String
+	selectedPapersAndPublications:[String!]!
```

Comment:
This probably shouldn't be a mandatory fields since now when we want to query for instructor profiles, we must provide this field in the search input.